joomla/joomla-cms
# [5.2] [Guided tours] Increase the width and adjust the height of the popup

### Summary of Changes

When creating tours, media content is very limited with the size of the popup.
Shepherd forces a popup width to be 400px max.
This PR proposes to increase the width to 600px and a popup that resizes according to its content.
In the case of a popup positioned a the center of the window, the max width could be increased even more (up to 1200px).

### Testing Instructions

Run a few tours and see if the popups resize properly with different window sizes.
Add an image to a tour, run the tour and look at the result, either in a central position or at a bottom position.

### Actual result BEFORE applying this Pull Request

![image](https://github.com/user-attachments/assets/f56e5507-00a2-49f9-aed5-b8c749770977)

![image](https://github.com/user-attachments/assets/99d70d10-6e17-4934-aad1-f85a10c9a7f9)

![image](https://github.com/user-attachments/assets/2724d9d9-fa63-449c-a9ed-228aa412c8c8)

### Expected result AFTER applying this Pull Request

The content dictates the width of the popups, up to 600px, except for the center popups, up to 1200px.

![image](https://github.com/user-attachments/assets/13839618-f8a6-4266-a09c-641ea791d99c)

![image](https://github.com/user-attachments/assets/a6839f1e-a259-4b83-9bb7-df480d5c7919)

![image](https://github.com/user-attachments/assets/6c4f0e72-51a0-444d-a38f-55a40bc1fdec)

### Link to documentations
Please select:
